po/es.po

```diff
@@ -6833,95 +6904,91 @@ msgid ""
 "}\n"
 "```"
 msgstr ""
-"```rust,editable\n"
-"fn main() {\n"
-"    let s1: String = String::from(\"Hola!\");\n"
-"    let s2: String = s1;\n"
-"    println!(\"s2: {s2}\");\n"
-"    // println!(\"s1: {s1}\");\n"
-"}\n"
-"```"
 
 #: src/ownership/copy-clone.md:30
 msgid "After the assignment, both `p1` and `p2` own their own data."
-msgstr "Luego de la asignación, ambos `p1` y `p2` obtienen su propio dato."
+msgstr ""
+"Después de la asignación, tanto `p1` como `p2` tienen sus propios datos."
 
 #: src/ownership/copy-clone.md:31
 msgid "We can also use `p1.clone()` to explicitly copy the data."
-msgstr "También podemos usar `p1.clone()` para explícitamente copiar datos."
+msgstr ""
+"También podemos utilizar `p1.clone()` para copiar los datos de forma "
+"explícita."
 
 #: src/ownership/copy-clone.md:35
 msgid "Copying and cloning are not the same thing:"
-msgstr "Copiar datos y clonar no es la misma cosa:"
+msgstr "Copiar y clonar no es lo mismo:"
 
 #: src/ownership/copy-clone.md:37
 msgid ""
 "Copying refers to bitwise copies of memory regions and does not work on "
 "arbitrary objects."
 msgstr ""
-"Copiar refiere a copias bit a bit de regiones de memoria y no funciona con "
-"objetos arbitrarios."
+"Copiar hace referencia a las copias bit a bit de regiones de memoria y no "
+"funciona en objetos aleatorios."
```

Comment:
"random" in spanish (aleatorio) very rarely can be used to mean "anything"/"your choice"/"any possibility". It happened to me once when using the word in the English way and I did not get understood.

The common way to word this is: "y no funciona en cualquier objeto."